roles/ovirt-image-template/README.md

```diff
@@ -28,6 +28,8 @@ Role Variables
 | template_disk_interface | virtio           | Interface of the template disk. |
 | template_timeout   | 600                   | Amount of time to wait for the template to be created. |
 | template_nics      | {name: nic1, profile_name: ovirtmgmt, interface: virtio} | List of dictionaries that specify the NICs of template. |
+| glance_image_provider        | UNDEF            | Name of the glance image provider.                    |
+| glance_image            | UNDEF               | this parameter specifies the name of disk to be imported as template. |
```

Comment:
This parameter specifies the name of disk in glance provider to be imported as template.